Examples/TicTacToe/Sources/Core/TwoFactorCore.swift

```diff
@@ -23,6 +23,10 @@ public enum TwoFactorAction: Equatable {
   case twoFactorResponse(Result<AuthenticationResponse, AuthenticationError>)
 }
 
+public struct TwoFactorTearDownToken: Hashable {
```

Comment:
This is for simplicity, but I imagine if you want to enumerate over all the cancel tokens you could do something like:

```swift
public enum TwoFactorTearDownToken: CaseIterable, Hashable {
  case twoFactorRequest
  case someOtherLongLivingEffect
}
```

And then in the parent:

```swift
return .merge(TwoFactorTearDownToken.allCases.map(Effect.cancel(id:)))
```

Reply:
I like the approach of using an enum - even if you aren't using it for the purposes of tearing things down, that seems preferable to having a large number of empty structs to handle multiple cancellations.